src/features/account/MoreOptionMenu.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { MoreOptionMenu } from './MoreOptionMenu';
+import { render } from '../../utils/test-utils';
+
+describe('MoreOptionMenu', () => {
+  it('matches the snapshot', () => {
```

Comment:
I would prefer testing the text or content of the component.

Reply:
have added a tests to test the content of this component.